src/Illuminate/Queue/composer.json

```diff
@@ -15,7 +15,9 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-PDO": "*",
```

Comment:
https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Queue/DatabaseQueue.php#L256-L280

Reply:
We shouldn't require the PDO extension for the entire Queue component as it's only used for the database queue. If I use the Redis queue for example I don't need PDO. IMO ext-PDO should only be in the suggest section.

Reply:
I agree. I'm going through now and making tweaks from required to suggested 👍

Still a WIP.